metadata.json

```diff
@@ -60,7 +60,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.6.1 < 5.0.0"
+      "version_requirement": ">= 4.7.0 < 5.0.0"
```

Comment:
IMO we should bump after we test against 5.X. We already discovered some problems on our zabbix module.